docs/entities/template.md

```
@@ -0,0 +1,116 @@
+  "lines": [...],
+  "layer": ["uuid-1", "uuid-2", "uuid-3"]
+}
+```
+
+`id`, `name`, `productVariantId`, `previewUrl` are separate DB columns, not in `data`.
+
+---
+
+## DB Table: `templates`
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)